### PR TITLE
Milestone 5: Bookstore styling

### PR DESCRIPTION
**Pull request by Linushaddai99**
In this branch, I improved the look and feel of the bookstore website by adding CSS styling.

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,9 +1,67 @@
...
   justify-content: space-between;
   align-items: center;
 }
 
+.navigation {
+  display: flex;
+  align-items: center;
+
+  /* width: 100%; */
+}
```
Comment (AmaduKamara) [inline src/App.css]: - [ ] Kindly remove unused codes from your codebase. It's a good practice to not have unused codes in your codebase as that will make your code clean and lean and professional. Please remove it so your codebase looks clean and professional 👍

File: src/App.css [hunk 2]
```diff
@@ -1,9 +1,67 @@
...
+.logo {
+  font-size: 1.875rem;
+  font-weight: 700;
+  color: #0290ff;
+}
+
+/* h1,
+h2 {
+  font-weight: lighter;
+} */
```
Comment (AmaduKamara) [inline src/App.css]: - [ ] Kindly remove unused codes from your codebase. It's a good practice to not have unused codes in your codebase as that will make your code clean and lean and professional. Please remove it so your codebase looks clean and professional 👍

File: src/App.css [hunk 3]
```diff
@@ -13,7 +71,188 @@ nav ul {
...
+  color: #0290ff;
+}
+
+.progress-section {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  /* width: 100%;
+  max-width: 200px; */
```
Comment (AmaduKamara) [inline src/App.css]: - [ ] Kindly remove unused codes from your codebase. It's a good practice to not have unused codes in your codebase as that will make your code clean and lean and professional. Please remove it so your codebase looks clean and professional 👍

File: src/App.css [hunk 4]
```diff
@@ -13,7 +71,188 @@ nav ul {
...
+  padding-right: 30px;
+  margin-right: 30px;
+
+  /* border: 1px solid chocolate; */
+}
```
Comment (AmaduKamara) [inline src/App.css]: - [ ] Kindly remove unused codes from your codebase. It's a good practice to not have unused codes in your codebase as that will make your code clean and lean and professional. Please remove it so your codebase looks clean and professional 👍

File: src/App.css [hunk 5]
```diff
@@ -13,7 +71,188 @@ nav ul {
...
+  margin: 0 30px;
+}
+
+.form-btn {
+  cursor: pointer;
+  width: 20%;
+  padding: 12px;
+  border-radius: 3px;
+  background-color: #0290ff;
+  border: none;
```
Comment (AmaduKamara) [inline src/App.css]: - [ ] Kindly work on the form fields and the button to stretch and align with the cards just as I highlighted in the screenshot below so things look aligned properly.

**Your Design**

image

**The Original Design**

image

File: src/App.css [hunk 6]
```diff
@@ -13,7 +71,188 @@ nav ul {
...
+  -webkit-transform: rotate(45deg);
+  transform: rotate(45deg);
+  margin-right: 15px;
+}
+
+.percentage {
+  font-family: "Montserrat", sans-serif;
+  color: #121212;
+  font-size: 2rem;
+}
```
Comment (AmaduKamara) [inline src/App.css]: - [ ] Currently, the progress percentages for several books are not aligned either on the left or right. Kingly works on their alignment either on the left as marked in the screenshot or on the right so your design matches the original design given. 👍 

**Your Design**

image

**The Original Design**

image

**Review by AmaduKamara — CHANGES_REQUESTED** [review]
# Status: Required Changes ♻️♻️♻️ 

Hi @Linushaddai99,

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✅ You used Gitflow correctly 👍 
✅ Linter checks passed 👍 
✅ App functionalities work fine as expected 👍 

## Required Changes ♻️
_Check the comments under the review._

- [ ] Kindly work on the design alignments so your design matched the original design. 👍 
- [ ] Kindly consider removing all commented codes from your codebase so your code is clean and professional 👍 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by frank1738 — APPROVED** [review]
Hi @Linushaddai99 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights

✅ Correct use of Gitflow
✅ No linter error
✅ App is designed as expected 👍 

## Optional suggestions

N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**